src/org/thoughtcrime/securesms/notifications/FcmReceiveService.java

```diff
@@ -81,6 +81,12 @@ public void onMessageReceived(@NonNull RemoteMessage remoteMessage) {
     // we just let the app run as long as possible.
   }
 
+  @Override
+  public void onDeletedMessages() {
```

Comment:
no call to super?

Reply:
this is not needed, it is just used as an event, neither Signal for the [example code](https://firebase.google.com/docs/cloud-messaging/android/receive) calls super